external/spring/build.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
```

Comment:
Same comment as for netty, just follow [derby](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/derby/build.xml) as an example